homework/index.js

```diff
@@ -0,0 +1,21 @@
+const repoSelectElement = document.getElementById('repositories');
+const unContainer = document.querySelector('.right__div-username');
+const reposContainer = document.querySelector('.right__div-repos');
+const avatarContainer = document.querySelector('.right__div-avatar');
+
+
+const CONTRIBUTORS_URL = async () => {
```

Comment:
You're creating an (async) function here (we might see that Sunday :) ). You should give it a function name. This kind of CAPITAL_CASE is only used for actual **constants**: values which are 'hard-coded' in code, usually at the top so they're easy to find and change, and then used throughout the code. As you know, functionNames should use pascalCasing in Javascript. Call this function getContributors. 

It will have to somehow get the contributors url to call; which is found in the repo object under the 'contributors_url' property (as hinted in week 1's guide). Either it will have to receive that as a parameter directly, or retrieve it from the (global) repositories array; in that case you might want to use the selected index.